modules/templates/helper.go

```diff
@@ -972,17 +972,27 @@ type remoteAddress struct {
 	Password string
 }
 
-func mirrorRemoteAddress(ctx context.Context, m *repo_model.Repository, remoteName string) remoteAddress {
+func mirrorRemoteAddress(ctx context.Context, m *repo_model.Repository, remoteName string, isIgnoreOriginalURL bool) remoteAddress {
```

Comment:
`isIgnoreOriginalURL` -> `ignoreOriginalURL`

Reply:
Your suggestions are better. Thank you for your modification suggestions. I have finished the adjustment.